models/category/src/category-draft/presets/sample-data-goodstore/cheese-trays.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import serveware from './serveware';
+
+const servewareDraft = serveware().build<TCategoryDraft>();
+
+const cheeseTrays = (): TCategoryDraftBuilder =>
+  empty()
+    .name(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('Cheese Trays')
+        ['en-US']('Cheese Trays')
+        ['de-DE']('Käseplatten')
+    )
+    .slug(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('cheese-trays')
+        ['en-US']('cheese-trays')
+        ['de-DE']('kaseplatten')
```

Comment:
The slug looks to be `cheese-trays` in the source data 🤔

Reply:
fixed with [439a32c](https://github.com/commercetools/test-data/pull/357/commits/439a32c559fb3c5e19eec43ab4b2a07dc184ff10)